### PR TITLE
Drop support for node 4

### PR DESCRIPTION
**Pull request by dgeb**

### CODE REVIEW

**Comment by stefanpenner** [comment]
can you also add the corresponding `engines` property to the projects `package.json` ?

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "4"
   - "6"
   - "stable"
```
Comment (rwjblue) [inline .travis.yml:6]:
```suggestion
  - "8"
  - "10"
  - "stable"
```

**Comment by dgeb** [comment]
@stefanpenner @rwjblue ok, thanks for the review - just updated

**Comment by stefanpenner** [comment]
LGTM

**Review by stefanpenner — APPROVED** [review]

**Pull request merged**